### ISSUE TITLE
SQL errors not thrown by PDOMySQL class

### ISSUE THREAD

**Issue by FaaPz** [devvoh/parable#12]
Several times my SQL queries (which are perfect thanks to Parable's query builder!) failed during execution. I've mistaken a unique index on one of my columns.

But... Parable never warned me! `Database::query()` simply returned `false`. Which is legit, according to the PHP docs [here](http://php.net/manual/en/pdo.query.php).

PDO errors are silenced _by default_ when the occur ([see PHP docs once more](http://php.net/manual/en/pdo.error-handling.php)). Please, make them load with warnings or exceptions (by "Parable default")! 😄

**Comment by FaaPz** [comment]
Oh! I've forgot to mention that it's not possible to access PDO's `errorCode()` and `errorInfo()` methods in the `Query` and `Repository` classes. 😜

**Comment by devvoh** [comment]
You make a good point! In 0.12.5, you'll be able to do the following:

```
"database" => [
    "type" => \Parable\ORM\Database::TYPE_MYSQL,
    "location"  => "localhost",
    "username"  => "username",
    "password"  => "password",
    "database"  => "database",
    "errorMode" => \PDO::ERRMODE_EXCEPTION,
],
```

Default remains `ERRMODE_SILENT`, though. Like a debug setting, I'd say throwing database-related errors should be a conscious choice. Obviously, documentation is essential with stuff like this 🙂

**Comment by devvoh** [comment]
Should be fixed in 0.12.5. Reopen if it requires fixes!
